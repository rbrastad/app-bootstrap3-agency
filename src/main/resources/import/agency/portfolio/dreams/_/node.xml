<node>
    <id>3566733f-a9a8-4969-9558-81d80513ff2b</id>
    <childOrder>modifiedtime DESC</childOrder>
    <nodeType>content</nodeType>
    <timestamp>2016-04-25T18:52:20.716Z</timestamp>
    <inheritPermissions>false</inheritPermissions>
    <permissions>
        <principal key="role:cms.cm.app">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:cms.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.everyone">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
    </permissions>
    <data>
        <boolean name="valid">true</boolean>
        <string name="displayName">Dreams</string>
        <string name="type">no.item.xp.starter.bootstrap3.agency:portfolio</string>
        <string name="owner">user:system:su</string>
        <dateTime name="modifiedTime">2016-04-17T06:49:33.702Z</dateTime>
        <string name="modifier">user:system:su</string>
        <string name="creator">user:system:su</string>
        <dateTime name="createdTime">2016-03-26T18:50:44.576Z</dateTime>
        <property-set name="data">
            <string name="title">Dreams</string>
            <string name="preface">Website Design</string>
            <string name="body">&lt;p&gt;Dreams is a free PSD web template built by Mathavan Jaya. Dreams is a modern one page web template designed for almost any purpose. It&amp;rsquo;s a beautiful template that&amp;rsquo;s designed with the Bootstrap framework in mind.&lt;/p&gt;
&lt;p&gt;You can download the PSD template in this portfolio sample item at FreebiesXpress.com.&lt;/p&gt;</string>
            <reference name="image">955d9dca-671e-439d-be1e-425c8a62aead</reference>
            <reference name="imagePreview">eed8e693-2ef2-4ca1-8201-c0a1eddf8afb</reference>
        </property-set>
        <property-set name="x">
            <property-set name="no-item-xp-starter-bootstrap3-agency">
                <property-set name="menu-item">
                    <boolean name="menuItem">false</boolean>
                    <string isNull="true" name="menuName"/>
                </property-set>
            </property-set>
        </property-set>
    </data>
    <indexConfigs>
        <analyzer>document_index_default</analyzer>
        <defaultConfig>
            <decideByType>true</decideByType>
            <enabled>true</enabled>
            <nGram>false</nGram>
            <fulltext>false</fulltext>
            <includeInAllText>false</includeInAllText>
        </defaultConfig>
        <pathIndexConfigs>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page.regions</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>true</nGram>
                    <fulltext>true</fulltext>
                    <includeInAllText>true</includeInAllText>
                    <indexValueProcessors>
                        <indexValueProcessor>htmlStripper</indexValueProcessor>
                    </indexValueProcessors>
                </indexConfig>
                <path>data.htmlarea_text</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifier</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>owner</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>attachment</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>x</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>true</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>data</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>site</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>type</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>creator</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>createdTime</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifiedTime</path>
            </pathIndexConfig>
        </pathIndexConfigs>
    </indexConfigs>
</node>
