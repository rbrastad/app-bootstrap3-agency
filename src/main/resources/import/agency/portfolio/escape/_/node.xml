<node>
    <id>5b330640-63a1-4b87-aec2-aa730497b00f</id>
    <childOrder>modifiedtime DESC</childOrder>
    <nodeType>content</nodeType>
    <timestamp>2016-04-25T18:52:20.712Z</timestamp>
    <inheritPermissions>false</inheritPermissions>
    <permissions>
        <principal key="role:cms.cm.app">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:cms.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.everyone">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
    </permissions>
    <data>
        <boolean name="valid">true</boolean>
        <string name="displayName">Escape</string>
        <string name="type">no.item.xp.starter.bootstrap3.agency:portfolio</string>
        <string name="owner">user:system:su</string>
        <dateTime name="modifiedTime">2016-04-17T06:50:05.703Z</dateTime>
        <string name="modifier">user:system:su</string>
        <string name="creator">user:system:su</string>
        <dateTime name="createdTime">2016-03-26T18:50:01.320Z</dateTime>
        <property-set name="data">
            <string name="title">Escape</string>
            <string name="preface">website design</string>
            <string name="body">&lt;p&gt;Escape is a free PSD web template built by Mathavan Jaya. Escape is a one page web template that was designed with agencies in mind. This template is ideal for those looking for a simple one page solution to describe your business and offer your services.&lt;/p&gt;
&lt;p&gt;You can download the PSD template in this portfolio sample item at FreebiesXpress.com.&lt;/p&gt;</string>
            <reference name="image">0037e5e2-5a62-490b-9c0f-d8a3222bbd91</reference>
            <reference name="imagePreview">0037e5e2-5a62-490b-9c0f-d8a3222bbd91</reference>
        </property-set>
        <property-set name="x">
            <property-set name="no-item-xp-starter-bootstrap3-agency">
                <property-set name="menu-item">
                    <boolean name="menuItem">false</boolean>
                    <string isNull="true" name="menuName"/>
                </property-set>
            </property-set>
        </property-set>
    </data>
    <indexConfigs>
        <analyzer>document_index_default</analyzer>
        <defaultConfig>
            <decideByType>true</decideByType>
            <enabled>true</enabled>
            <nGram>false</nGram>
            <fulltext>false</fulltext>
            <includeInAllText>false</includeInAllText>
        </defaultConfig>
        <pathIndexConfigs>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page.regions</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>true</nGram>
                    <fulltext>true</fulltext>
                    <includeInAllText>true</includeInAllText>
                    <indexValueProcessors>
                        <indexValueProcessor>htmlStripper</indexValueProcessor>
                    </indexValueProcessors>
                </indexConfig>
                <path>data.htmlarea_text</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifier</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>owner</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>attachment</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>x</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>true</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>data</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>site</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>type</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>creator</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>createdTime</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifiedTime</path>
            </pathIndexConfig>
        </pathIndexConfigs>
    </indexConfigs>
</node>
