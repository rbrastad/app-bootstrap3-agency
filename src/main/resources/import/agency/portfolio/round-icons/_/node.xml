<node>
    <id>9ae05bbb-7ca3-4900-bac4-ad5f8015ddf7</id>
    <childOrder>modifiedtime DESC</childOrder>
    <nodeType>content</nodeType>
    <timestamp>2016-04-25T18:52:20.719Z</timestamp>
    <inheritPermissions>false</inheritPermissions>
    <permissions>
        <principal key="role:cms.cm.app">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:cms.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.everyone">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
    </permissions>
    <data>
        <boolean name="valid">true</boolean>
        <string name="displayName">Round Icons</string>
        <string name="type">no.item.xp.starter.bootstrap3.agency:portfolio</string>
        <string name="owner">user:system:su</string>
        <dateTime name="modifiedTime">2016-04-17T06:49:00.832Z</dateTime>
        <string name="modifier">user:system:su</string>
        <string name="creator">user:system:su</string>
        <dateTime name="createdTime">2016-03-26T09:28:39.624Z</dateTime>
        <property-set name="data">
            <string name="title">Round Icons</string>
            <string name="preface">Graphic Design</string>
            <string name="body">&lt;p&gt;Use this area to describe your project. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Est blanditiis dolorem culpa incidunt minus dignissimos deserunt repellat aperiam quasi sunt officia expedita beatae cupiditate, maiores repudiandae, nostrum, reiciendis facere nemo!&lt;/p&gt;
&lt;p&gt;Want these icons in this portfolio item sample?You can download 60 of them for free, courtesy ofRoundIcons.com, or you can purchase the 1500 icon set here.&lt;/p&gt;
&lt;p&gt;&amp;nbsp;&lt;/p&gt;</string>
            <reference name="image">6077b486-7224-4185-bbc8-cf2cf436caff</reference>
            <reference name="imagePreview">9d81018e-45cc-4bbb-a670-b4b60c7e452d</reference>
        </property-set>
        <property-set name="x">
            <property-set name="no-item-xp-starter-bootstrap3-agency">
                <property-set name="menu-item">
                    <boolean name="menuItem">false</boolean>
                    <string isNull="true" name="menuName"/>
                </property-set>
            </property-set>
        </property-set>
    </data>
    <indexConfigs>
        <analyzer>document_index_default</analyzer>
        <defaultConfig>
            <decideByType>true</decideByType>
            <enabled>true</enabled>
            <nGram>false</nGram>
            <fulltext>false</fulltext>
            <includeInAllText>false</includeInAllText>
        </defaultConfig>
        <pathIndexConfigs>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page.regions</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>true</nGram>
                    <fulltext>true</fulltext>
                    <includeInAllText>true</includeInAllText>
                    <indexValueProcessors>
                        <indexValueProcessor>htmlStripper</indexValueProcessor>
                    </indexValueProcessors>
                </indexConfig>
                <path>data.htmlarea_text</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifier</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>owner</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>attachment</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>x</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>true</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>data</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>site</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>type</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>creator</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>createdTime</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifiedTime</path>
            </pathIndexConfig>
        </pathIndexConfigs>
    </indexConfigs>
</node>
