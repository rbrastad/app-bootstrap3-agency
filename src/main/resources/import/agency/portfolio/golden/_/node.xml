<node>
    <id>111da410-3db9-4979-b8f0-cc8e14d37202</id>
    <childOrder>modifiedtime DESC</childOrder>
    <nodeType>content</nodeType>
    <timestamp>2016-04-25T18:52:20.708Z</timestamp>
    <inheritPermissions>false</inheritPermissions>
    <permissions>
        <principal key="role:cms.cm.app">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:cms.admin">
            <allow type="array">
                <value>READ</value>
                <value>CREATE</value>
                <value>MODIFY</value>
                <value>DELETE</value>
                <value>PUBLISH</value>
                <value>READ_PERMISSIONS</value>
                <value>WRITE_PERMISSIONS</value>
            </allow>
            <deny type="array"/>
        </principal>
        <principal key="role:system.everyone">
            <allow type="array">
                <value>READ</value>
            </allow>
            <deny type="array"/>
        </principal>
    </permissions>
    <data>
        <boolean name="valid">true</boolean>
        <string name="displayName">Golden</string>
        <string name="type">no.item.xp.starter.bootstrap3.agency:portfolio</string>
        <string name="owner">user:system:su</string>
        <dateTime name="modifiedTime">2016-04-17T06:50:27.462Z</dateTime>
        <string name="modifier">user:system:su</string>
        <string name="creator">user:system:su</string>
        <dateTime name="createdTime">2016-03-26T18:48:43.921Z</dateTime>
        <property-set name="data">
            <string name="title">Golden</string>
            <string name="preface">Website Design</string>
            <string name="body">&lt;p&gt;Start Bootstrap's Agency theme is based on Golden, a free PSD website template built by Mathavan Jaya. Golden is a modern and clean one page web template that was made exclusively for Best PSD Freebies. This template has a great portfolio, timeline, and meet your team sections that can be easily modified to fit your needs.&lt;/p&gt;
&lt;p&gt;You can download the PSD template in this portfolio sample item at FreebiesXpress.com.&lt;/p&gt;</string>
            <reference name="image">1c052534-b1a8-4db9-9712-a7c56e407d88</reference>
            <reference name="imagePreview">c7c95be9-d170-4668-984f-380127428899</reference>
        </property-set>
        <property-set name="x">
            <property-set name="no-item-xp-starter-bootstrap3-agency">
                <property-set name="menu-item">
                    <boolean name="menuItem">false</boolean>
                    <string isNull="true" name="menuName"/>
                </property-set>
            </property-set>
        </property-set>
    </data>
    <indexConfigs>
        <analyzer>document_index_default</analyzer>
        <defaultConfig>
            <decideByType>true</decideByType>
            <enabled>true</enabled>
            <nGram>false</nGram>
            <fulltext>false</fulltext>
            <includeInAllText>false</includeInAllText>
        </defaultConfig>
        <pathIndexConfigs>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page.regions</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>true</nGram>
                    <fulltext>true</fulltext>
                    <includeInAllText>true</includeInAllText>
                    <indexValueProcessors>
                        <indexValueProcessor>htmlStripper</indexValueProcessor>
                    </indexValueProcessors>
                </indexConfig>
                <path>data.htmlarea_text</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifier</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>owner</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>attachment</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>x</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>true</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>data</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>page</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>false</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>site</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>type</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>creator</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>createdTime</path>
            </pathIndexConfig>
            <pathIndexConfig>
                <indexConfig>
                    <decideByType>false</decideByType>
                    <enabled>true</enabled>
                    <nGram>false</nGram>
                    <fulltext>false</fulltext>
                    <includeInAllText>false</includeInAllText>
                </indexConfig>
                <path>modifiedTime</path>
            </pathIndexConfig>
        </pathIndexConfigs>
    </indexConfigs>
</node>
